lib/twilio/device.ts

```diff
@@ -1840,6 +1843,12 @@ namespace Device {
      */
     appVersion?: string;
 
+    /**
+     * The callMessage types a client subscribes to. 'user-defined-message' will always be opted in by
+     * default.
+     */
```

Comment:
>An array of [call message types](https://twilio.github.io/twilio-voice.js/interfaces/voice.call.message.html#messagetype) to subscribe to. Currently, only 'user-defined-message' is supported. More message types will be added in the future. See [call resource](https://www.twilio.com/docs/voice/api/call-resource) documentation for more details.
>
>When subscribed, the SDK will emit the message via [messageReceivedEvent](https://twilio.github.io/twilio-voice.js/classes/voice.call.html#messagereceivedevent).

Reply:
I used markdown style for the links. Please update the links for call message types and messageReceivedEvent to use typdoc/tsdoc style.